### PR TITLE
Add feature to give the option to copy clean files to new bucket.

### PR DESCRIPTION
**Pull request by CptEzz**
If the AV_DEFINITION_S3_CLEAN_BUCKET environment is specified, if a scanned file is marked as CLEAN it will then copy the file to the new bucket.

### CODE REVIEW

**Review by RoryStokes — APPROVED** [review]
Looks good to me

**Review by SamMurphyDev — APPROVED** [review]
lgtm

File: scan.py [hunk 1]
```diff
@@ -149,6 +150,16 @@ def set_av_tags(s3_client, s3_object, scan_result, scan_signature, timestamp):
         Bucket=s3_object.bucket_name, Key=s3_object.key, Tagging={"TagSet": new_tags}
     )
 
+def move_clean_file(s3, s3_object):
```
Comment (NoxHarmonium) [inline scan.py]: Does this move the file or copy it?

**Review by NoxHarmonium — CHANGES_REQUESTED** [review]
Maybe the function names could be changed to copy instead of move to be clearer (unless I misunderstand) but apart from that LGTM

**Review by CptEzz — COMMENTED** [review]
I have addressed the concerns around the naming of the functions

**Review by NoxHarmonium — APPROVED** [review]

**Pull request merged**